### ISSUE TITLE
0.6 Fails to Launch on elementary Hera 5.1.7 (Ubuntu 18.04 LTS)

### ISSUE THREAD

**Issue by chances** [HungryProton/concept_graph#97]
After downloading the 0.6.2-preview release, the executable fails to launch:

> $ ./ConceptGraph                                          
./ConceptGraph: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by ./ConceptGraph)

My system has glibc `v2.27`.

As a workaround, are there instructions for building the standalone app from source?

**Comment by HungryProton** [comment]
Yes, you can find the manual instructions there: https://github.com/HungryProton/concept_graph_environment
If you don't want to recompile a custom version of the engine, there's binaries hosted on the repository but it might not work either. 

Releases hosted here are creating using Github Actions running on Ubuntu latest. With a bit of luck, maybe changing this to 16.04 would be enough?

**Comment by chances** [comment]
Or maybe the Ubuntu build could use the 14.04 [docker image](https://hub.docker.com/_/ubuntu)?

**Comment by chances** [comment]
Also, I think I'm mistaken about Godot using 16.04. Their Actions [workflow file for Linux](https://github.com/godotengine/godot/blob/3.2/.github/workflows/linux_builds.yml) targets 20.04.

**Comment by HungryProton** [comment]
I need to check how much would need to change to use the docker image, I'm still new at CI in general.

By the way, it seems they stopped using 14.04 for the official builds as well according to their workflow file?
https://github.com/godotengine/godot/blob/3.2/.github/workflows/linux_builds.yml
